package.json

```diff
@@ -58,11 +58,12 @@
     "mocha": "^2.2.5",
     "object-assign": "^3.0.0",
     "phantomjs": "^1.9.17",
-    "react": "^0.14.2",
-    "react-addons-test-utils": "^0.14.2",
-    "react-dom": "^0.14.2",
+    "prop-types": "^15.5.8",
```

Comment:
As of React v15.5.0--seriously--you can't use `React.PropTypes` and instead have to use this standalone library